tycho-surefire/org.eclipse.tycho.surefire.osgibooter/src/main/java/org/eclipse/tycho/surefire/osgibooter/OsgiSurefireBooter.java

```diff
@@ -217,6 +220,46 @@ private static Map<String, String> extractProviderProperties(Properties surefire
         return providerProps;
     }
 
+    private static SurefireConsoleOutputReporter getSurefireConsoleOutputReporter(String tychoReportingMode) {
+        if (tychoReportingMode.equals("JUnit5")) {
```

Comment:
I'm not sure if there is any convention for Maven POM properties, but I find it rather confusing to have a property value that is not lowercase or camelCase. This will become a source of error for users, therefore I would just write it as "junit5" in the documentation instead.
Independent of the actual predefined value, wouldn't it make sense to compare case insensitive here?

Reply:
Thanks for the comment. Yes, if we actually do need to add a new parameter, lowercase would probably be better.